scripts/quests/bastok/Beauty_And_The_Galka.lua

```diff
@@ -0,0 +1,151 @@
+-----------------------------------
+-- Beauty and the Galka
+-----------------------------------
+-- !addquest 1 1
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require("scripts/globals/items")
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_util")
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+
+local portBastokIDs = require("scripts/zones/Port_Bastok/IDs")
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.BASTOK, xi.quest.id.bastok.BEAUTY_AND_THE_GALKA)
+
+quest.reward =
+{
+    fame = 75,
+    fameArea = xi.quest.fame_area.BASTOK,
+    item = xi.items.BRONZE_KNIFE
+}
+
+quest.sections =
+{
+    -- Quest Acceptance 1
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE
+        end,
+
+        [xi.zone.PORT_BASTOK] =
+        {
+            ['Talib'] = quest:progressEvent(2),
+
+            onEventFinish =
+            {
+                [2] = function(player, csid, option, npc)
+                    if option == 1 then
+                        quest:setVar(player, 'Denied', 1)
+                        return
+                    end
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    -- Quest Acceptance 1a
+    {
+        check = function(player, status, vars)
+            return
+                status == QUEST_AVAILABLE and
+                quest:getVar(player, 'Denied') == 1
```

Comment:
can do vars.Denied == 1. However, we try to stick to only using the vars: Prog, Option, Stage. This is purely to help GMs debug quests for players and makes it much easier for them.

Reply:
Prog, Option, Stage is good, but wouldn't a more useful term be.. more useful to a GM when looking at the logs/vars for a quest?  The cause of the charvar to be set is the player Denying the quest.

Reply:
yes. that can be argued. but this is our code style and it is how our GMs access variables.

Reply:
Changed to Option as it's the most relevant.